fr/development/routing.rst

```diff
@@ -830,7 +830,7 @@ utilisés lors de la connexion des routes. Par exemple, si votre action
 renommer vos actions::
 
     $routes->resources('Articles', [
-        'actions' => ['update' => 'put', 'create' => 'create']
+        'actions' => ['update' => 'put', 'add' => 'create']
```

Comment:
@Pontax are you ok with @HavokInspiration proposal? If yes, could you also make the change for english version ?